File: backend/src/main/java/com/team05/petmeeting/domain/feed/dto/FeedReq.java
package com.team05.petmeeting.domain.feed.dto;

import com.team05.petmeeting.domain.feed.enums.FeedCategory;

public record FeedReq(
        FeedCategory category,
        String title,
        String content,
        String imageUrl,
        Long animalId
) {
}

File: backend/src/main/java/com/team05/petmeeting/domain/feed/dto/FeedReq.kt
package com.team05.petmeeting.domain.feed.dto

import com.team05.petmeeting.domain.feed.enums.FeedCategory

data class FeedReq(
    val category: FeedCategory,
    val title: String,
    val content: String,
    val imageUrl: String?,
    val animalId: Long?
)